scripts/drag-n-bar.js

```diff
@@ -1117,18 +1118,17 @@ H5P.DragNBar.prototype.addControlBoxOnElement = function (element) {
     
     // Adding control-box
     const startStringControlBox = 'h5p-control-box-unique-';
-    const uniqueControlBox = startStringControlBox + uniqueClassFloat.toString(32);
-    self.createMoveableControlBoxOnElement(element.$element, uniqueControlBox);
+    const uniqueControlBoxId = startStringControlBox + uniqueClassFloat.toString(32);
+    self.createMoveableControlBoxOnElement(element.$element, uniqueControlBoxId);
 
     // Hiding moveable-control-boxes. This is because if we edit a whole CP (from the menu), we dont want all the boxes to show.
     // If we are just adding an element, it will get 'focused' after this code is run, so it's ok.
     self.hideControlBoxes();
 
-    // Moving the control-box to overlap the element.
-    const rectElementBCR = element.$element[0].getBoundingClientRect();
-    const theControlBoxElement = document.getElementsByClassName(uniqueControlBox)[0];
-    const dndToolbarHeight = 38;
-    theControlBoxElement.style.transform = `translate3d(${rectElementBCR.left + window.scrollX}px, ${rectElementBCR.top + window.scrollY + dndToolbarHeight}px, 0px)`;
+    // Since the context-menu-box will be added after the control-box, it will push the element down, but not the control-box-element.
+    // Therefore, we are adjusting the control-box's position on the element 'after' the context-menu has pushed the element down by using requestAnimationFrame(),
+    // so that the control-box is exactly overlapping the element.
+    requestAnimationFrame(() => self.adjustControlBoxPositionOnElement(element, uniqueControlBoxId));
```

Comment:
There's no reason for aliasing `this` here.

```suggestion
    requestAnimationFrame(() => this.adjustControlBoxPositionOnElement(element, uniqueControlBoxId));
```